packages/mobile/src/geth/saga.ts

```diff
@@ -137,19 +137,22 @@ export function* initGethSaga() {
     }
   }
 
-  CeloAnalytics.track(CustomEventNames.geth_init_failure, {
-    error: result,
+  ValoraAnalytics.track(AnalyticsEvents.geth_init_failure, {
+    error: result.message,
```

Comment:
Looks like `result` is just of type GethInitOutcomes and can be passed without .message